lib/agent0/examples/example_agent.py

```diff
@@ -24,6 +24,9 @@
 
 # Build custom policy
 # Simple agent, opens a set of all trades for a fixed amount and closes them after
+# TODO this bot is almost identical to the one defined in test_fixtures for system tests
```

Comment:
Where is this bot being used? My vote is to remove it. In the docs we can point to the cycle_trade_policy for people to see the trades.

Reply:
This bot is being used as an example (i.e., this script calls the bot). It's meant as a template for writing custom bots.

Reply:
I see, well what about simplifying it, then? Like have it return a no-op (empty list) or have it open a long and then immediately close it. We don't need it to complete every possible trade to demonstrate how to write a custom agent. And if it's super long then people might be weary of making it work. The random agent policy shows all trades, and the new test policy shows all trades. So people have examples.